pyiele/config.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+class config:
+  port = 8342
+  passphrase = "walletNotSecure"
+  master_key = "m-test-shl-mk1ypc6mjyustmuma5vf2kvxm2aml2clfewfmk7mvkuk8344vdxzgwseytxyg"
+  gas_price = "0x0"
+  gas_limit = "0x166f5777"
+  target_directory = "contract_artifacts"
```

Comment:
Is this file passed as an argument, or is it hardcoded right now? I guess it would be nice if users could pass it as an argument?

Reply:
Most of the values used in the config class can be overwritten by values passed by the user as command line arguments.